### PR TITLE
Large post

### PR DESCRIPTION
**Pull request by JimmyCYJ**
Add test that verifies PR#[1318](https://github.com/istio/proxy/pull/1318) fixes #[1315](https://github.com/istio/proxy/issues/1315).

### CODE REVIEW

File: mixer/test/client/check_report_large_post_request/check_report_large_post_request_test.go [hunk 1]
```diff
@@ -109,11 +103,11 @@ func TestCheckReportLargePostRequest(t *testing.T) {
 	}
 	defer s.TearDown()
 
+	// Issues a POST request with 10 MB request body. This request is sent to ServerProxyPort
+	// directly. This verifies that the Mixer filter at ingress Envoy could handle large request.
```
Comment (qiwzhang) [inline mixer/test/client/check_report_large_post_request/check_report_large_post_request_test.go]: please add TODO:  use ClientProxyPort after issuer https://github.com/envoyproxy/envoy/issues/2929 is fixed,

File: mixer/test/client/check_report_large_post_request/check_report_large_post_request_test.go [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+  "quota.cache_hit": false,
+  "request.headers": {
+     ":method": "POST",
+     ":path": "/echo",
+     ":authority": "*",
+     "x-forwarded-proto": "http",
+     "x-istio-attributes": "-",
+     "x-request-id": "*"
+  },
+  "request.size": "*",
```
Comment (qiwzhang) [inline mixer/test/client/check_report_large_post_request/check_report_large_post_request_test.go]: can we verify request.size and response.size?

**Comment by JimmyCYJ** [comment]
Please take a look. Thanks!

**Comment by linsun** [comment]
thanks for getting this in, /lgtm

File: mixer/test/client/check_report_large_post_request/check_report_large_post_request_test.go [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+	s := env.NewTestSetup(env.CheckReportLargePostRequestTest, t)
+	env.SetStatsUpdateInterval(s.MfConfig(), 1)
+	if err := s.SetUp(); err != nil {
+		t.Fatalf("Failed to setup test: %v", err)
+	}
+	defer s.TearDown()
+
+	// Issues a POST request with 10 MB request body. This request is sent to ServerProxyPort
+	// directly. This verifies that the Mixer filter at ingress Envoy could handle large request.
+	// TODO: use ClientProxyPort after issuer envoyproxy/envoy#2929 is fixed.
```
Comment (wattli) [inline mixer/test/client/check_report_large_post_request/check_report_large_post_request_test.go]: Please add your or someone else's github id after TODO.

File: mixer/test/client/env/ports.go [hunk 4]
```diff
@@ -27,6 +27,7 @@ const (
 	CheckCacheTest uint16 = iota
 	CheckCacheHitTest
 	CheckReportAttributesTest
+	CheckReportLargePostRequestTest
```
Comment (wattli) [inline mixer/test/client/env/ports.go]: Sort?

File: mixer/test/client/check_report_large_post_request/check_report_large_post_request_test.go (hunk 3 above)
Comment (JimmyCYJ) [reply mixer/test/client/check_report_large_post_request/check_report_large_post_request_test.go]: Done.

File: mixer/test/client/env/ports.go (hunk 4 above)
Comment (JimmyCYJ) [reply mixer/test/client/env/ports.go]: Done.

**Comment by JimmyCYJ** [comment]
Please take a look. Thanks!

**Pull request merged**